Source/ReferenceConnectorWorkerService/ConnectorTask.cs

```diff
@@ -154,20 +154,25 @@ private async Task SubmitItem(CancellationToken cancellationToken)
                 // Something went wrong trying to submit the item. 
                 // Dead-letter the item to a durable data store where it can be retried later. (e.g., a message broker).
             }
-            
-            // After submitting, check to see if Records365 vNext has a record of the parent item that was referenced
-            // in the submission above. If it doesn't, we need to submit it.
-            if (submitContext.AggregationFoundDuringItemSubmission.HasValue &&
-                !submitContext.AggregationFoundDuringItemSubmission.Value)
+

```

Comment:
I'm pretty sure we need to pull the `Filters` off the ConnectorConfig and put it on the `SubmitContext` as well - I think @0shi can confirm